rwengine/src/engine/GameWorld.hpp

```diff
@@ -401,6 +396,26 @@ class GameWorld {
 
     PlayerController* getPlayer();
 
+    template <
+        typename T1, typename T2 = T1,
+        typename std::enable_if<std::is_integral<T1>::value>::type* = nullptr,
+        typename std::enable_if<std::is_integral<T2>::value>::type* = nullptr>
+    T1 getRandomNumber(T1 min, T2 max) {
```

Comment:
static?

Reply:
`static` is impossible due to the random source (`randomNumberGen`).
I would rename the function to `getRandomUniform`. But that's a detail.